care/facility/api/serializers/facility.py

```diff
@@ -75,7 +79,7 @@ class FacilitySerializer(FacilityBasicInfoSerializer):
     #     "latitude": 49.8782482189424,
     #     "longitude": 24.452545489
     # }
-    read_cover_image_url = serializers.CharField(read_only=True)
```

Comment:
lets keep this as a Charfield

Reply:
Lets store file name and domain for the bucket separately

Reply:
converting it to URLfield will fix the api schema

Reply:
My bad, i thought it was a model change